configurations/rvi_pipeline_configuration.py

```diff
@@ -124,7 +124,7 @@
                     engagement_db_dataset="rvi_elections_s01e01",
                     code_scheme_configurations=[
                         CodeSchemeConfiguration(
-                            code_scheme=load_code_scheme("s01e01"),
+                            code_scheme=load_code_scheme("rqas/rvi/s01e01"),
```

Comment:
```suggestion
                            code_scheme=load_code_scheme("rqas/rvi_elections/s01e01"),
```

Reply:
update also in other sections

Reply:
Thanks, I mustn't have tested this specific case after the rename. I've renamed and tested both the pre-existing configurations in analysis and in coda sync to ensure that all the code schemes load now.